### ISSUE TITLE
Loss values/graphs for pretrained models on LJSpeech

### ISSUE THREAD

**Issue by oytunturk** [fatchord/WaveRNN#68]
Do you happen to have the loss values for pretrained tacotron and wavernn models? I started training my models from scratch. Losses seem to be decreasing as expected but it would be nice to know when they typically start to level off on LJSpeech corpus.

Thanks for the repo and for sharing the pretrained models!

**Comment by fatchord** [comment]
@oytunturk I'm not sure how useful the loss is as a metric with these models. In fact tacotron will often overfit if left train long enough - at least with small datasets.

**Comment by oytunturk** [comment]
Yes, that's right. Anyway, training from scratch seems to work fine for me. When you trained the WaveRNN model, did you wait until Tacotron model training completion and then generate gta data for WaveRNN or did you do generate gta data in some intermediate Tacotron training iteration? I think waiting for Tacotron completion is likely to provide better training/test match hence better WaveRNN performance but it might be a minor quality difference after a certain point. I wonder if you have any insights...
